File: src/reference/webhooks/verify.cs
// Using asp.net style requests, you may need to adjust it :)
public bool IsRequestValid(IHttpContext context, string secret, string body) {
    var hmac = new HMACSHA384(Encoding.UTF8.GetBytes(secret));
    var hash = BitConverter.ToString(hmac.ComputeHash(Encoding.UTF8.GetBytes(body))).Replace("-", string.Empty);
    return context.HttpContext.Request.Headers["X-Poker-Signature"].Equals($"sha384={hash}");
}

File: src/reference/webhooks/verify.py
import hmac

# Using Flask-style requests, you may need to adjust it :)
def is_request_valid(request, body: str, secret: str):
    hm = hmac.new(bytes(secret, 'utf-8'), digestmod='SHA384')
    hm.update(bytes(body, 'utf-8'))
    expected = 'sha384=' + hm.hexdigest().upper())
    return hmac.compare_digest(request.headers['poker-signature'], expected)
